zipline/finance/slippage.py

```diff
@@ -95,7 +107,9 @@ def __repr__(self):
                    volume_limit=self.volume_limit,
                    price_impact=self.price_impact)
 
-    def process_order(self, bar_data, price, volume, order):
+    def process_order(self, data, order):
+        price = data.current(order.asset, "price")
```

Comment:
The volume check is done one level up in the call stack, so `process_order` will never be called.

https://github.com/quantopian/zipline/pull/1064/files#diff-7553035d28cf2d71ff96e833ebb225c3R52